order/supplyunit.go

```diff
@@ -10,7 +10,7 @@ type SupplyUnit uint64
 const (
 	// BaseSupplyUnit is the smallest channel that can be bought or sold in
 	// the system. These units are expressed in satoshis.
-	BaseSupplyUnit SupplyUnit = 100_000
+	BaseSupplyUnit btcutil.Amount = 100_000
```

Comment:
Why change? To cutdown on casting somewhere else?

Reply:
It just seemed strange to define it as `SupplyUnit` as you could then invoke `ToSatoshis` on it resulting in `100000^2`.